libdnf/dnf-repo.c

```diff
@@ -869,11 +865,18 @@ dnf_repo_set_keyfile_data(DnfRepo *repo, GError **error)
     }
 
     /* gpgkey is optional for gpgcheck=1, but required for repo_gpgcheck=1 */
-    g_free(priv->gpgkey);
-    priv->gpgkey = g_key_file_get_string(priv->keyfile, priv->id, "gpgkey", NULL);
+    g_strfreev(priv->gpgkeys);
+    tmp_strval = g_key_file_get_string(priv->keyfile, priv->id, "gpgkey", NULL);
+    if (tmp_strval) {
+        priv->gpgkeys = g_strsplit_set(tmp_strval, " ,", -1);
```

Comment:
i.e. it can be either space or comma separated.  We also use this strsplit pattern for the `exclude` processing.  I didn't do newlines though...hopefully that won't be needed?  I'm not sure the glib parser would handle that even.